tools/automation.py

```diff
@@ -0,0 +1,217 @@
+    '''
+    # Mentors' data saved as a list of dictionaries
+    mentors = []
+
+    # Load data from the Mentors sheet into a DataFrame object
+    df_excel = pd.read_excel(xlsx_file, sheet_name="Mentors")
+
+    for i in range(0, len(df_excel)):
+        # Get table row
+        print(f"Current index: {i}")
+        mentor_row = df_excel.iloc[i]
+
+        # Save strings as yaml multiline strings
+        mentee_str = get_multiline_string(mentor_row.iloc[24])
+        bio_str = get_multiline_string(mentor_row.iloc[25])
+        extra_str = get_multiline_string(mentor_row.iloc[26])
+
+        # Save mentor's areas, focus and programming languages data in a list format
+        # List format converts to block sequences in yaml file
+        # Block sequences indicate each entry with a dash and space ("- ")
+        areas = get_data_sequence_as_list(mentor_row,
+                                            AREAS_START_INDEX,
+                                            AREAS_END_INDEX)
+        focus = get_data_sequence_as_list(mentor_row,
+                                            FOCUS_START_INDEX,
+                                            FOCUS_END_INDEX)
+        programming_languages = get_data_sequence_as_list(mentor_row,
+                                                            PROG_LANG_START_INDEX,
+                                                            PROG_LANG_END_INDEX)
+
+        network_list = get_social_media_links(mentor_row.iloc[27])
+        hours_per_month = get_numbers_from_string(mentor_row.iloc[28])
+        max_experience = get_numbers_from_string(mentor_row.iloc[8])
+
+        # TODO: Temp solution - change
+        disabled = False
+        matched = True
+        sort = 10
+        index = i+1
+        image = ""
+
+        mentor = {'name': mentor_row.iloc[0],
+                'disabled': disabled,
+                'matched': matched,
+                'sort': sort,
+                'hours': hours_per_month,
+                'type': mentor_row.iloc[2],
+                'index': index,
+                'location': mentor_row.iloc[4],
+                'position': f"{mentor_row.iloc[6]}, {mentor_row.iloc[7]}",
+                'bio': bio_str,
+                'image': image,
+                'languages': mentor_row.iloc[5],
+                'skills':{
+                    'experience': mentor_row.iloc[8],
+                    'years': max_experience,
+                    'mentee': mentee_str,
+                    'areas': areas,
+                    'languages': ', '.join(programming_languages),
+                    'focus': focus,
+                    'extra': extra_str,
+                    },
+                'network': network_list,
+                }
+        mentors.append(mentor)
+    return mentors
+
+if __name__ == "__main__":
+    print(os.getcwd())
+    FILE_PATH = ".\\tools\\mentors.xlsx"
+    list_of_mentors = read_mentors_xlsx_file(FILE_PATH)
+    create_mentors_yml_file(list_of_mentors)
```

Comment:
I would like to suggest two options: 
1 - Create the complete file (read all mentors and generate output)  (it could be this version 1 you did, and we can improve later) 
2 - Update exist file with from last x lines, for example: 
Update current yml from last 2 mentors: we are going to read the existent file and append into the end the `x` new mentors.